core/utils.ts

```diff
@@ -72,7 +72,7 @@ export {
  * Halts the propagation of the event without doing anything else.
  *
  * @param e An event.
- * @deprecated
+ * @deprecated This is no longer needed.
```

Comment:
Maybe change to "no longer provided by Blockly".

Reply:
Updated!